### ISSUE TITLE
About input of text

### ISSUE THREAD

**Issue by TitaniumOne** [KaiyangZhou/CoOp#50]
Thanks for your great job!
I want to ask why the input is not (image, text) at forward function, such as `output = self.model(image, text)` . 
And what is the scheme of matching text logits and image logits?
